### PR TITLE
git*: add Italian translation

### PR DESCRIPTION
**Pull request by glenacota**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.it/common/git-send-email.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+`git send-email -1 {{commit}}`
+
+- Invia 10 commit nel ramo corrente:
+
+`git send-email {{-10}}`
+
+- Invia un'email con un messaggio introduttivo alla serie di patch:
+
+`git send-email -{{numero di commit}} --compose`
```
Comment (navarroaxel) [inline pages.it/common/git-send-email.md:21]: We use snake_case for variable names:
```suggestion
`git send-email -{{numero_di_commit}} --compose`
```
Looks an issue with the English page :disappointed:

File: pages.it/common/git-send-email.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+- Revisiona e modifica il messaggio email per ogni patch da inviare:
+
+`git send-email -{{numero di commit}} --annotate`
```
Comment (navarroaxel) [inline pages.it/common/git-send-email.md:25]:
```suggestion
`git send-email -{{numero_di_commit}} --annotate`
```

**Review by navarroaxel — APPROVED** [review]

File: pages.it/common/git-send-email.md (hunk 1 above)
Comment (glenacota) [reply pages.it/common/git-send-email.md:21]: I could work on a bulk change, as there are other cases (e.g., buku, gist, man, ...)

**Review by navarroaxel — APPROVED** [review]

**Comment by glenacota** [comment]
Thanks to all the reviewers :) if there are no strong objections, I'm going to squash and merge in a couple of hours

**Comment by glenacota** [comment]
I'm afraid I can't merge it myself. @sbrl, could you please complete the review of this PR? :) thanks!

**Review by schneiderl — APPROVED** [review]

**Comment by glenacota** [comment]
Thanks, everyone! :)

**Pull request merged**